runtime/interpreter/simplecompositevalue.go

```diff
@@ -55,6 +57,11 @@ func NewSimpleCompositeValue(
 	fieldFormatters map[string]func(Value, SeenReferences) string,
 	stringer func(SeenReferences) string,
 ) *SimpleCompositeValue {
+	inter.UseMemory(common.MemoryUsage{
+		Kind:   common.MemoryKindComposite,
+		Amount: uint64(1),
```

Comment:
cast to `uint64` is redundant for literals
```suggestion
		Amount: 1,
```